webgrid/tests/test_filters.py

```diff
@@ -85,6 +84,37 @@ def def_val():
         self.assert_in_query(query, "WHERE persons.firstname LIKE '%bar%'")
 
 
+class TestTextFilterCaseSensitive(CheckFilterBase):
```

Comment:
These test cases seem to check for case insensitivity, contra the name of the test class